pygmt/datasets/load_remote_dataset.py

```diff
@@ -84,11 +80,10 @@ class GMTRemoteDataset(NamedTuple):
             "01m": Resolution("01m", registrations=["gridline"], tiled=True),
         },
     ),
-    "earth_free_air_anomaly": GMTRemoteDataset(
-        title="free air anomaly",
-        name="free_air_anomaly",
-        long_name="IGPP Earth Free-Air Anomaly",
-        units="mGal",
+    "earth_mag": GMTRemoteDataset(
```

Comment:
Please also move the `earth_mag` entry below the `earth_mask` entry.

Reply:
The alphabetical order was updated in https://github.com/GenericMappingTools/pygmt/pull/3048/commits/ab00c939bcb6ae21b972a52efefea6337e33d5d6.